### ISSUE TITLE
Virus infection on SourceForge!?

### ISSUE THREAD

**Issue by Excaliburst** [winpython/winpython.github.io#18]
Hi I just downloaded newest versions of WinPython 3.3, & 3.4 (32bit) from Sourceforce, but my Antivirus Avast interrupted both downloads with the reason that the files were infected with the Win32:Evo-gen virus.

Can anyone confirm that the Source-Forge version is Virus free?

Thanks

**Comment by sebix** [comment]
Please post the exact version and/or the link to the files you downloaded.

**Comment by Excaliburst** [comment]
http://sourceforge.net/projects/winpython/files/WinPython_3.3/3.3.5.5/WinPython-32bit-3.3.5.5.exe/download

http://sourceforge.net/projects/winpython/files/WinPython_3.4/3.4.2.4/WinPython-32bit-3.4.2.4.exe/download

**Comment by stonebig** [comment]
did you check your md5 / Sha-1 on arrival ?

		MD5			|	SHA-1 | Program 
------------------------------ | ---------------------- | -------------------
f74b2d8e5f6dd94921ee3e892e0c3622 | 28ddc164e02502ffda21dc84f93568396f38f44d | winpython-32bit-3.4.2.4.exe

**Comment by stonebig** [comment]
My personnal download experience on an different PC with Symantec up-to-date protection:
- did let me download,
- did provide the same binary file (md5 / sha-1) as I have uploaded 10 days ago,
- a scan of the binary didn't provoque any alarm,
- and you are the first to alarm.

==> As you couldn't download, we can't say if the alarm of Avast was on this same binary.
==> Try download  3.3.5.0 version for example. If there is the same alarm, I will suspect it's a false one, given the you would be the first to complain after 12 696 downloads

**Comment by stonebig** [comment]
Anyway, thank you for having spent time to hask for a check. 
I hope another person with yet another different anti-virus will check also.

**Comment by Excaliburst** [comment]
Hi Stonebig,

I did not get the file in its entirety as Avast stopped the DL after 40MB out of 180MB if I remember correctly. So Checksum does not do me any good.

So to proceed I did however 
- Disable Avast during the DL. Reenable it after. 
- Check the full file for virus.
- And then check the checksum

Result: With your MD5 checksum for the file
http://sourceforge.net/projects/winpython/files/WinPython_3.4/3.4.2.4/WinPython-32bit-3.4.2.4.exe/download
MD5: f74b2d8e5f6dd94921ee3e892e0c3622

I get identical checksum. So I believe my Avast was spoofed for second.

Thank you for taking time to reassure me that my Avast might be wrong in its alarm. And for your tips on checking MD checksum.

But as you know; our PCs are our livelyhood... So better safe than sorry!
